plugins/server/io.github.flaxoos/ktor-server-task-scheduling/versions.ktor.yaml

```
@@ -0,0 +1,5 @@
+"[2.0,)":
+  - io.github.flaxoos:ktor-server-task-scheduling-core-jvm:1.2.9+
+  - io.github.flaxoos:ktor-server-task-scheduling-redis-jvm:1.2.9+
+  - io.github.flaxoos:ktor-server-task-scheduling-mongodb-jvm:1.2.9+
+  - io.github.flaxoos:ktor-server-task-scheduling-jdbc-jvm:1.2.9+
```

```suggestion
"[2.0,)":
  - io.github.flaxoos:ktor-server-task-scheduling-core-jvm:$flaxoosExtraPlugins
  - io.github.flaxoos:ktor-server-task-scheduling-redis-jvm:$flaxoosExtraPlugins
  - io.github.flaxoos:ktor-server-task-scheduling-mongodb-jvm:$flaxoosExtraPlugins
  - io.github.flaxoos:ktor-server-task-scheduling-jdbc-jvm:$flaxoosExtraPlugins

flaxoosExtraPlugins: 1.+
```

Variables were just recently added.  This will create a `flaxoos_extra_plugins_version` property in the resulting `gradle.properties` and import it in the `build.gradle.kts`.  You can also share the variables between plugins, like the Kafka one.  Soon, we'll add project template support for version catalogs and Amper which will use these variables as well.